fe/fe-core/src/main/java/com/starrocks/catalog/OlapTable.java

```diff
@@ -2984,20 +2985,24 @@ private boolean isEnableFillDataCacheImpl(Partition partition) throws AnalysisEx
             RangePartitionInfo rangePartitionInfo = (RangePartitionInfo) getPartitionInfo();
             Range<PartitionKey> partitionRange = rangePartitionInfo.getRange(partition.getId());
             Range<PartitionKey> dataCacheRange;
-            if (rangePartitionInfo.isPartitionedBy(PrimitiveType.DATETIME)) {
-                LocalDateTime upper = LocalDateTime.now();
-                LocalDateTime lower = upper.minus(cacheDuration);
-                dataCacheRange = Range.openClosed(PartitionKey.ofDateTime(lower), PartitionKey.ofDateTime(upper));
-                return partitionRange.isConnected(dataCacheRange);
-            } else if (rangePartitionInfo.isPartitionedBy(PrimitiveType.DATE)) {
-                LocalDate upper = LocalDate.now();
-                LocalDate lower = upper.minus(cacheDuration);
-                dataCacheRange = Range.openClosed(PartitionKey.ofDate(lower), PartitionKey.ofDate(upper));
-                return partitionRange.isConnected(dataCacheRange);
-            } else {
-                // If the table was not partitioned by DATE/DATETIME, ignore the property "datacache.partition_duration" and
-                // enable data cache by default.
-                return true;
+            try {
+                if (rangePartitionInfo.isPartitionedBy(PrimitiveType.DATETIME)) {
+                    LocalDateTime upper = LocalDateTime.now();
+                    LocalDateTime lower = upper.minus(cacheDuration);
+                    dataCacheRange = Range.openClosed(PartitionKey.ofDateTime(lower), PartitionKey.ofDateTime(upper));
+                    return partitionRange.isConnected(dataCacheRange);
+                } else if (rangePartitionInfo.isPartitionedBy(PrimitiveType.DATE)) {
+                    LocalDate upper = LocalDate.now();
+                    LocalDate lower = upper.minus(cacheDuration);
+                    dataCacheRange = Range.openClosed(PartitionKey.ofDate(lower), PartitionKey.ofDate(upper));
+                    return partitionRange.isConnected(dataCacheRange);
+                } else {
+                    // If the table was not partitioned by DATE/DATETIME, ignore the property "datacache.partition_duration" and
+                    // enable data cache by default.
+                    return true;
+                }
+            } catch (UnsupportedTemporalTypeException e) {
+                LOG.error(e.getMessage());
```

Comment:
1. you need to comment when this exception will be thrown.
2. change LOG.error to LOG.warn, error often be used for system errors.

Reply:
OK